### PR TITLE
Ipek task sdep 5535

### PR DESCRIPTION
**Pull request by ipekyilmaz35**
test

### CODE REVIEW

File: cypress/e2e/shopping/buyProducts.e2e.cy.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { CheckoutPage } from '../../support/views/purchasing/checkoutPage'
 import { OverviewPage } from '../../support/views/purchasing/overviewPage'
 import { FinalPage } from '../../support/views/purchasing/finalPage'
 
-describe('User can buy products from page, simplest happy path end to end scenario', () => {
+describe('User can buy products from page, --', () => {
   const loginPage: LoginPage = new LoginPage()
```
Comment (kwaiga8) [inline cypress/e2e/shopping/buyProducts.e2e.cy.ts:10]: Please remove "--" from the end

File: cypress/e2e/shopping/cartHandling.cy.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { MainShopPage } from '../../support/views/inventory/inventory'
 import { LoginPage } from '../../support/views/login/loginPage'
 import { Item } from '../../support/locators/inventory/inventory'
 
-describe('User can add and remove items to the card on the shop inventory page', () => {
+describe('User can add and remove items to the card in the shop inventory page', () => {
```
Comment (kwaiga8) [inline cypress/e2e/shopping/cartHandling.cy.ts]: Change to: "User can add and remove items to the card"

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # better-cypress
 <p>
 Yes you can write elegant code in Cypress + TypeScript!
-<p>Java + Selenium is the best option for automated UI tests? Let's try smth ligther ; )
+<p>Java + Selenium is the best option for automated UI tests? Let's try something lighter ; )
```
Comment (kwaiga8) [inline README.md]: I would change to: "faster and lighter"

**Review by kwaiga8 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by ipekyilmaz35** [comment]
I have made the changes you mentioned for these files: readme, buyProducts, cartHandling

**Comment by ipekyilmaz35** [comment]
> Please address the comments

okay

**Review by kwaiga8 — APPROVED** [review]
Thats nice!

**Pull request merged**